app/pages/apps.vue

```diff
@@ -21,5 +21,7 @@ let filteredApps = computed(() => {
 </template>
 
 <style scoped>
-
+  * {
+    color: black;
```

Comment:
since the project uses Tailwind, please use it where possible!

see for color
https://github.com/ToBinio/project-manager/blob/7de20702f6b8d8fd74602fc02d1a74d9147bfd7e/app/layouts/default.vue#L6